docs/learn-bridges.md

```diff
@@ -4,54 +4,105 @@ title: Bridges
+
+Bridge modules will be written with particular chains in mind such as Bitcoin
+or Ethereum. This means that blockchains that are derivative of these will
+likely be easily supported as well (e.g. Litecoin and other Bitcoin-forks).
+A parachain
+
+The bridging modules will likely be written with particular parachains in mind 
+(e.g. Bitcoin, Ethereum), which means that any blockchain that's based on either 
+of those should be able to be bridged directly to Polkadot without the need of 
+going through a bridge contract on a parachain. This should allow for faster 
+execution for those chains that are compatible.
+
+For the standalone chains that will not have a parachain bridging module on 
+Polkadot, it will be necessary to deploy bridge contracts (see below).
+
+## Bridge contracts
+
+Those who are already familiar with Ethereum may know of the [Parity Bridge][]
+and the efforts being made to connect PoA sidechains to the Ethereum mainnet. 
+The bridge is a combination of two smart contracts, one deployed on each chain, 
+that allow for cross-chain transfers of value. As an example of usage, the 
+initial Parity Bridge proof of concept connects two Ethereum chains, `main` and 
+`side`. Ether deposited into the contract on `main` generate a balance 
+denominated in ERC-20 tokens on `side`. Conversely, ERC-20 tokens deposited back 
+into the contract on `side` can free up Ether on `main`.
+
+In the case of Polkadot, it should be possible to have a bridge contract deployed 
+on, say, an EVM-based standalone chain and a contract deployed on a smart contract 
+capable parachain. This would not necessarily be the most efficient method of 
+bridging, but given the generality of a Turing-complete parachain it would be 
+possible to bridge Polkadot and any other smart contract capable blockchain.
+
+## Bitcoin Bridge
+
+The Interlay team has written a [specification][interlay] on a Bitcoin bridge
+that is based on the [XClaim][] design paper. The protocol enables a two-way
+bridge between Polkadot and Bitcoin. It allows holders of BTC to "teleport"
+their assets to Polkadot as PolkaBTC, and holders of PolkaBTC to burn their
+assets for BTC on the Bitcoin chain.
+
+The Bitcoin bridge as documented in the specification is composed of two
+logically different components:
+
+- The XCLAIM component that maintains all accounts that own PolkadBTC.
+- The BTC-Relay that is responsible for verifying Bitcoin state when a new
+transaction is submitted.
+
+For full details on how it works please refer to the specification.
+
+## Ethereum Bridge
+
+As explained by Dr. Gavin Wood in a [blog post][eth bridging blog] from late 2019,
+there are three ways that the Polkadot and Substrate ecosystem can be bridged to
+the Ethereum ecosystem.
+
+1) Polkadot <-> Ethereum Public Bridge.
+1) Substrate <-> Parity Ethereum (Openethereum) Bridge.
+1) The Substrate EVM module.
+
+Please read the blog article for fuller descriptions of each one of these
+options.
 
 ## Resources
```

Comment:
Do we maybe want to end with a CTA for teams to apply for grants if they want to build bridges?

Reply:
good idea